e2e-tests/production-runtime/cypress/integration/assets.js

```diff
@@ -11,18 +11,37 @@ describe(`webpack assets`, () => {
   if (!Cypress.env(`TEST_PLUGIN_OFFLINE`)) {
     it(`should only create one font file (no duplicates with different hashes)`, () => {
       // Check that there is no duplicate files (should have italic as second request, not another normal font)
-      cy.wait("@font").should(req => {
-        expect(req.response.url).to.match(/merriweather-latin-300-/i)
-      })
-      cy.wait("@font").should(req => {
-        expect(req.response.url).to.match(/merriweather-latin-300italic-/i)
-      })
+      let filesNeeded = [
+        new RegExp(`merriweather-latin-300-`, 'i'),
+        new RegExp(`merriweather-latin-300italic-`, 'i'),
```

Comment:
Would it make more sense to setup different interceptors instead of single?
```diff
-cy.intercept("/static/merriweather-latin-300**.woff2").as("font")
+cy.intercept("/static/merriweather-latin-300-**.woff2").as("font-regular")
+cy.intercept("/static/merriweather-latin-300italic-**.woff2").as("font-italic")
```
So that we can wait on both before doing assertions?

Reply:
🤦 Yeah, I like that approach a lot more.